tests/integration/customerNotes.test.js

```diff
@@ -0,0 +1,195 @@
+  });
+});
+
+describe('CUSTOMER NOTES ROUTES - POST /customernotes', () => {
+  let authToken;
+  beforeEach(populateDB);
+
+  describe('AUXILIARY', () => {
+    beforeEach(async () => {
+      authToken = await getToken('auxiliary');
+    });
+
+    it('should create a customer note', async () => {
+      const response = await app.inject({
+        method: 'POST',
+        url: '/customernotes',
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload: { title: 'Titre', description: 'description', customer: customersList[0]._id },
+      });
+
+      expect(response.statusCode).toBe(200);
+    });
+
+    it('should return 400 if missing customer', async () => {
+      const response = await app.inject({
+        method: 'POST',
+        url: '/customernotes',
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload: { title: 'Titre', description: 'description' },
+      });
+
+      expect(response.statusCode).toBe(400);
+    });
+
+    it('should return 400 if missing title', async () => {
+      const response = await app.inject({
+        method: 'POST',
+        url: '/customernotes',
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload: { description: 'description', customer: customersList[0]._id },
+      });
+
+      expect(response.statusCode).toBe(400);
+    });
+
+    it('should return 400 if missing description ', async () => {
+      const response = await app.inject({
+        method: 'POST',
+        url: '/customernotes',
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload: { title: 'Titre', customer: customersList[0]._id },
+      });
+
+      expect(response.statusCode).toBe(400);
+    });
+
+    it('should return 400 if customer has invalid type ', async () => {
+      const response = await app.inject({
+        method: 'POST',
+        url: '/customernotes',
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload: { title: 'Titre', description: 'description', customer: '123423' },
+      });
+
+      expect(response.statusCode).toBe(400);
+    });
+
+    it('should return 400 if title has invalid type ', async () => {
+      const response = await app.inject({
+        method: 'POST',
+        url: '/customernotes',
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload: { title: 12342, description: 'description', customer: customersList[0]._id },
+      });
+
+      expect(response.statusCode).toBe(400);
+    });
+
+    it('should return 400 if description has invalid type ', async () => {
+      const response = await app.inject({
+        method: 'POST',
+        url: '/customernotes',
+        headers: { Cookie: `alenvi_token=${authToken}` },
+        payload: { title: 'Titre', description: 12345, customer: customersList[0]._id },
+      });
+
+      expect(response.statusCode).toBe(400);
+    });
+
+    it('should return 404 if customer has wrong company', async () => {
```

Comment:
que penses tu d'homogénéiser ici par rapport a celui d;en dessous `should return 404 if customer and user have different companies` ?

Reply:
done